staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4012,7 +4012,7 @@ type ServiceSpec struct {
 	// of the service will fail. This field can not be changed through updates.
 	// Valid values are "None", empty string (""), or a valid IP address. "None"
 	// can be specified for headless services when proxying is not required.
-	// Only applies to types ClusterIP, NodePort, and LoadBalancer. Ignored if
+	// Only applies to types ClusterIP, NodePort, and LoadBalancer. Wiped if
 	// type is ExternalName.
```

Comment:
should describe the exact circumstances under which it is cleared (e.g. "IP values are cleared on update from non-ExternalName service to ExternalName service if the value ClusterIP value remains unchanged")

Reply:
OK, a bunch of these comments can be updated.  Will do and re-push.